feature-libs/product-configurator/common/components/configure-product/configure-product.component.html

```diff
@@ -12,7 +12,7 @@
       "
       class="btn btn-primary btn-block"
       cxAutoFocus
-      >{{ 'configurator.header.toconfig' | cxTranslate }}</a
+      (keyup)="navigateToConfigurator($event, product)">{{ 'configurator.header.toconfig' | cxTranslate }}</a
```

Comment:
After tabbing on the product detail page util the `Configure` button and clicking `Space` key to enter the configuration, the product detail page scrolls down before entering the configuration. This is not a case when one enters the configuration via `Enter` key.
No scrolling is observed by reaching out `Add to cart` button neither via `Space` nor via `Enter` keys.

Reply:
Fixed.